build.gradle

```diff
@@ -50,6 +60,10 @@ compileTestJava {
     options.encoding = 'UTF-8'
     options.compilerArgs << "-Xlint:all"
 }
+compileIntegTestJava {
+    options.encoding = 'UTF-8'
+    options.compilerArgs << "-Xlint:all"
+}
```

Comment:
Good catch.  However, we can reduce the duplication here by moving this common configuration for `compileJava`, `compileTestJava`, and `compileIntegTestJava` to the `tasks.withType(JavaCompile)` block a few lines below.